src/app/pages/login/login-page/login.component.ts

```diff
@@ -35,8 +37,20 @@ export class LoginComponent implements OnInit {
 
   isLoggedIn = false;
 
+  returnUrl: string;
+
   ngOnInit() {
-    if (this.tokenStorage.getToken()) {
+    this.returnUrl =
+      this.activatedRoute.snapshot.queryParamMap.get('returnUrl');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/SimpleSurveyProject/SimpleSurvey-Frontend/pullRequest?prid=7430210)
```suggestion
      this.activatedRoute.snapshot.queryParamMap.get("returnUrl");
```